src/main/java/dev/blaauwendraad/masker/json/config/JsonMaskingConfig.java

```diff
@@ -119,13 +119,15 @@ public Map<String, KeyMaskingConfig> getKeyConfigs() {
 
     @Override
     public String toString() {
-        return "JsonMaskingConfig{" +
-               "targetKeys=" + targetKeys +
-               ", targetKeyMode=" + targetKeyMode +
-               ", caseSensitiveTargetKeys=" + caseSensitiveTargetKeys +
-               ", defaultConfig=" + defaultConfig +
-               ", targetKeyConfigs=" + targetKeyConfigs +
-               '}';
+        return """
+                JsonMaskingConfig{\n
+                targetKeys=%s,\n
+                targetKeyMode=%s,\n
+                caseSensitiveTargetKeys=%s,\n
+                defaultConfig=%s,\n
+                targetKeyConfigs=%s\n
+                }"""
```

Comment:
I don't think you need `\n` now, does it not create two line breaks? Also I think `"""` needs to be on the new line so that indent is calculated correctly

Reply:
Lemme print it to check and I'll fix.

Reply:
Updated this as well